nucleus/__init__.py

```diff
@@ -899,7 +901,23 @@ def _make_request(
         )
         logger.info("API request has response code %s", response.status_code)
 
-        if response.status_code != 200:
+        return response
+
+    def _make_request(
+        self, payload: dict, route: str, requests_command=requests.post
+    ) -> dict:
+        """
+        Makes a request to Nucleus endpoint and logs a warning if not
+        successful.
+
+        :param payload: given payload
+        :param route: route for the request
+        :param requests_command: requests.post, requests.get, requests.delete
+        :return: response JSON
+        """
+        response = self._make_request_raw(payload, route, requests_command)
+
+        if response.status_code not in [200, 201]:
```

Comment:
maybe factor out the list of "success" response codes to the constants.py file so that the variable can be reused everywhere.